src/v2/components/ProfileFollows/fragments/groupProfileFollowers.ts

```diff
@@ -8,8 +8,8 @@ export default gql`
     __typename
     id
     followers(page: $page, per: $per) {
-      ... IdentifiableCell
-      ... KonnectableCell
+      ...IdentifiableCell
+      # ...KonnectableCell
```

Comment:
@dzucconi - in order to get type generation to work I had to comment out this validation error:

```
Fragment "KonnectableCell" cannot be spread here as objects of type "User" can never be of type "Konnectable".
```

I'm pretty sure this will break something somewhere, but did seem like a legit error. Thoughts? 

This also occurs in `src/v2/components/ProfileFollows/fragments/userProfileFollowers.ts`.